docs/designers-developers/developers/block-api/block-patterns.md

```diff
@@ -23,6 +23,7 @@ register_block_pattern(
 	'my-plugin/my-awesome-pattern',
 	array(
 		'title'       => __( 'Two buttons', 'my-plugin' ),
+		'categories'  => array( 'my-plugin' ),
```

Comment:
Thanks again for the Pull Request! 

The value(s) inside the categories array needs to be an existing block pattern category. 

Although 'my-plugin' could be its own block category, a custom block pattern needs to be registered first and for the sake of simplicity for this documentation; we should use one of the default categories, which would be buttons. 

And then wrap it in a translation function, because buttons can mean some different context, so it would look like: 

`array( _x( 'buttons', 'Block pattern category', 'text-domain' )),`